engine/class_modules/monk/sc_monk.cpp

```diff
@@ -1976,7 +1990,18 @@ struct blackout_kick_t : charred_passions_t<monk_melee_attack_t>
 
       p()->buff.transfer_the_power->trigger();
 
-      p()->buff.teachings_of_the_monastery->expire();
+      if ( p()->buff.teachings_of_the_monastery->up() )
+      {
+        p()->buff.teachings_of_the_monastery->expire();
+
+        if ( p()->rng().roll( p()->talent.conduit_of_the_celestials.xuens_guidance->effectN( 1 ).percent() ) )
+          p()->buff.teachings_of_the_monastery->trigger();
+      }
+
+      if ( p()->specialization() == MONK_WINDWALKER )
+      {
+        p()->buff.strength_of_the_black_ox->expire();
+      }
```

Comment:
Strength of the Black Ox seems to be `on_expire` of a Xuen summon, not a buff, for WW.